Core/Extensions/ModelConversion/ModelConversionExtensions.cs

```diff
@@ -48,5 +48,35 @@ public static UpdateMemberCommand ToUpdateMemberCommand(this MemberVm model)
             };
             return command;
         }
+
+        public static CreateTaskCommand ToCreateTaskCommand(this TaskVm model)
+        {
+            var command = new CreateTaskCommand()
+            {
+                Subject = model.Subject,
+                AssignedMemberId = model.AssignedMemberId
+            };
+            return command;
+        }
+
+        public static AssignTaskCommand ToAssignTaskCommand(this TaskVm model)
+        {
+            if (model.Member?.Id == null) throw new Exception("No member was found to assign to.");

```

Comment:
Let's not throw general Exception here since we have more suitable `ArgumentNullException` here:
```
if (model.Member?.Id == null)
            {
                throw new ArgumentNullException(nameof(TaskVm.Member.Id), "No member was found to assign to.");
            }
```